import-dashboards.py

```diff
@@ -242,6 +242,13 @@ def set_home_dashboard(api_key):
     r = requests.put('%s/api/user/preferences' % (HOST,), data=data, headers=grafana_headers(api_key))
     print ' * Preferences set: %r %r' % (r.status_code, r.content)
 
+    # Copy pmm logo to the grafana directory
+    logo_file = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
+    dest_dir = '/usr/share/grafana/public/img/'
```

Comment:
For me `logo_file` and `dest_dir` looks like constants. Maybe it's better to move them to the head of the file and uppercase them?
What do you think about this?

Reply:
Done.